custom-recipes/wordcloud-recipe/recipe.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+import dataiku
+from dataiku.customrecipe import *
+import logging
+
+logger = logging.getLogger(__name__)
+
+import matplotlib
+import matplotlib.pyplot as plt
+
+matplotlib.use("agg")
+
+import traceback
+from typing import List, AnyStr
+import pandas as pd
+import spacy.lang
+import os
+from wordcloud import WordCloud
+from collections import Counter
+from time import time
+from spacy_tokenizer import MultilingualTokenizer
+from plugin_io_utils import count_records
+import random
+from wordcloud_generator import WordcloudGenerator
+
+
+# Load config
```

Comment:
I would rather centralize the "loading config" stuff in a separate module in Python lib. You can check this example: https://github.com/dataiku/dss-plugin-nlp-preparation/blob/master/python-lib/plugin_config_loading.py

It's important to add more parameter validation with custom error messages.